client/app/components/Input/InputTag.jsx

```diff
@@ -12,6 +12,7 @@ export type Props = {
   placeholder?: string,
   checkboxes: Checkbox[],
   onChange?: Function,
+  onCheckboxChange?: Function
```

Comment:
Just to be consistent with linting, let's add a `,` at the end of what's on line 15.